app/src/main/java/com/mobileproto/hireddit/hireddit/speech/WordToSpeech.java

```diff
@@ -41,6 +41,12 @@ public void speak(String toSpeak)
         }
     }
 
+    public void pause() {
+        if (mTtobj.isSpeaking()) {
+            mTtobj.stop();
+        }
+    }
+
     public void stop()
```

Comment:
Basically it is analogous to an deconstructor for an class, we need to free the resources used by the object in onDestroy()